OrbitBase/include/OrbitBase/Platform/Windows/Profiling.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_BASE_PLATFORM_WINDOWS_PROFILING_H_
+#define ORBIT_BASE_PLATFORM_WINDOWS_PROFILING_H_
+
+#include <Windows.h>
+
+#include <string>
+
+using pid_t = uint32_t;
```

Comment:
I dislike this a lot. `pid_t` is a Linux-specific name and it's very confusing that it's suddenly available in Windows code. These `typedef`s to bring one name to the other platform are bad practice. Even worse, on Linux `pid_t` is a signed type.